tests/openwisp2/settings.py

```diff
@@ -48,9 +48,16 @@
     'drf_yasg',
     'django_extensions',
     'openwisp2.integrations',
+    'djangosaml2',
```

Comment:
have you tested it without the djangosaml2 library installed in the virtual env and without this line? Does the app keep running?

Reply:
Yes! I added djangosaml2 in requirements-test.txt. Without it, tests were failing.